packages/vector_graphics_compiler/test/parser_test.dart

```diff
@@ -567,14 +579,16 @@ void main() {
     final VectorInstructions instructions = parseWithoutOptimizers(
       svg,
     );
-
     expect(instructions.paints, const <Paint>[
       Paint(fill: Fill(color: Color(0xffff0000))),
+      Paint(blendMode: BlendMode.screen, fill: Fill(color: Color(0xff000000))),
```

Comment:
Deleting the opacity peephole optimizer. It's not accounting for blend modes correctly.

We separately have an issue where blends are not getting respected correctly for some reason, but this doesn't seem to make anything worse.